warehouse/query-core/src/main/java/datawave/query/config/LookupUUIDTune.java

```diff
@@ -75,12 +74,9 @@ public void configure(BaseQueryLogic<Entry<Key,Value>> logic) {
             if (reduceResponse) {
                 rsq.setParseTldUids(true);
                 
-                // setup SeekingQueryPlanner in case the queryIterator requires it
-                SeekingQueryPlanner planner = new SeekingQueryPlanner();
-                planner.setMaxFieldHitsBeforeSeek(maxFieldHitsBeforeSeek);
-                planner.setMaxKeysBeforeSeek(maxKeysBeforeSeek);
-                
-                rsq.setQueryPlanner(planner);
+                // pass through seek options
+                rsq.setFiFieldSeek(maxFieldHitsBeforeSeek);
+                rsq.setFiNextSeek(maxKeysBeforeSeek);
```

Comment:
`maxFieldHitsBeforeSeek` and `maxKeysBeforeSeek` should be removed along with any getters/setters. 

Then you can set the fi and event seek values from lines 34-37